identification-bff.yml

```diff
@@ -6,22 +6,62 @@ servers:
   - url: http://localhost:8080
     description: Generated server url
 paths:
-  /api/v1/identification-bff/customers?page=[page]&size=[size]:
+  /api/v1/identification-bff/customers?pageNo=[pageNo]&pageSize=[pageSize]&firstName=[firstName]&lastName=[lastName]&userName=[userName]&legalName=[legalName]&commonName=[commonName]&status=[status]&address=[address]&customBroker=[customBroker]:
     get:
       tags:
         - customer-controller
       operationId: getCustomers
       parameters:
         - in: query
-          name: page
+          name: pageNo
           required: false
           schema:
             type: integer
         - in: query
-          name: size
+          name: pageSize
           required: false
           schema:
             type: integer
+        - in: query
+          name: firstName
+          required: false
+          schema:
+            type: string
+        - in: query
+          name: lastName
+          required: false
+          schema:
+            type: string
+        - in: query
+          name: userName
+          required: false
+          schema:
+            type: string
+        - in: query
```

Comment:
I thought we that would be one field searching if 2 columns in the database. It could be something like business name.